### ISSUE TITLE
Use the shared travis config with auto-updates for node releases

### ISSUE THREAD

**Issue by dominykas** [nodejs/core-validate-commit#85]
tl;dr: I'd like to open a PR to replace the `node_js` section in the `.travis.yml` with an imported https://github.com/nodejs/ci-config-travis/blob/master/lts/gte-8.yml

Background: [Package Maintenance WG](https://github.com/nodejs/package-maintenance) has built a repository of [shared Travis CI imports](https://github.com/nodejs/ci-config-travis). The imports will be automatically updated as new Node.js versions get released.

Judging by what's inside the current `.travis.yml` here, and the missing `engines` in the `package.json`, I suspect that the `lts` upgrade policy is the right fit (it adds new versions as soon as they are released, never removes LTS versions, removes non-LTS versions when they reach EOL).

Is that OK?

**Comment by richardlau** [comment]
I have no objections. I'd also be open to moving from Travis over to GitHub Actions like we have done for several other repositories in the nodejs org.

**Comment by dominykas** [comment]
Opened #86 - as for GH actions - keep an eye out for https://github.com/nodejs/ci-config-github-actions (once we get that figured out...)
